server/cpho/views/indicators.py

```diff
@@ -48,6 +60,17 @@ class Meta:
     )
 
 
+class UploadForm(forms.Form):
+    csv_file = forms.FileField(
+        required=True,
+        widget=forms.FileInput(
+            attrs={
+                "class": "form-control",
+            }
+        ),
+    )
+
```

Comment:
move the validation stuff from your form_valid into a `def clean_csv_file()` here. You can raise ValidationError rather than use messages